tests/test-httpModule.js

```diff
@@ -70,7 +70,8 @@ function runTests (name, httpModules) {
   tape(name, function (t) {
     var toHttps = 'http://localhost:' + plainServer.port + '/to_https'
     var toPlain = 'https://localhost:' + httpsServer.port + '/to_plain'
-    var options = { httpModules: httpModules, strictSSL: false }
+    var options = { httpModules: httpModules, strictSSL: false, allowInsecureRedirect: true }
+    var optionsSecure = { httpModules: httpModules, strictSSL: false }
```

Comment:
`optionsSecure` is never actually used in the test, is something missing here or is this a leftover?

Reply:
Good catch! It is a leftover. Ultimately I moved this check to other tape (in the test-redirect.js fileO